packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -11,13 +11,14 @@ import {
   position,
   flexbox,
 } from 'styled-system';
-import { ThemeProps, Theme } from '@t3n/theme';
+import { ThemeProps } from '@t3n/theme';
 import { Box } from '../Box';
 import { Text } from '../Text';
 import { Input } from '../Input';
 import { Button } from '../Button';
 import { Grid } from '../Grid';
 import { GridItem } from '../GridItem';
+import useIsMobile from '../helper/useIsMobile';
```

Comment:
Magst du das vlt noch in einen Ordner namens `hooks` auslagern, oberhalb von `helpers`? Dann könnte man früher oder später vielleicht noch eine Sammlung von Hooks rausgeben. Oder evtl ein eigenes Package draus machen (zukünftig, nicht jetzt).